tldw_Server_API/app/services/mcp_hub_external_access_resolver.py

```
@@ -323,6 +347,25 @@ async def _build_server_rows(
             )
         return rows
 
+    async def _managed_secret_ref_ready(self, managed_secret_ref_id: Any) -> bool:
```

Ruff (0.15.6)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `managed_secret_ref_id`

(ANN401)